Model/Currency.php

```diff
@@ -22,8 +22,9 @@ class Currency extends \Magento\Framework\Model\AbstractModel
     /**
      *  @var \Psr\Log\LoggerInterface
      */
-    protected $_logger; 
+    protected $_logger;
 
+    const PRECISION = 2;
```

Comment:
Created this ticket: https://withreach.atlassian.net/browse/MAG-109 to address my concerns round this value being hardcoded. Totally understand why it's hardcoded at this point in time.

Reply:
It is hard coded as per your recommendation  in MAG- 100 :
"There is a quick solution we could get to (always rounding to 2 decimal places) or the absolute correct way to do it (always round to the number of units that the selected currency supports, but no more than 2 digits due to a bug in our API)"

I can definitely take the longer if that is what we want.

Reply:
yeah! This is perfect for right now, just wanted to record the extra work of making precision dynamic for the future. I'd rather get this change out and come back to not having this value be hardcoded than slow things down by getting that work done now if that makes sense.